File: RUBY/Cap06/c06ex01.rb
# c06ex01.rb

#!/usr/bin/env ruby

md = Array.new(8)
soma = 0

for i in 0..7
  print "Entre a media do aluno #{i + 1}: "
  md[i] = gets.chomp.to_f
  soma += md[i]
end

media = soma / 8.0
puts "\nMedia geral: #{media.round(2)}"

print "\nTecle <Enter> para encerrar... "
gets

File: PYTHON/Cap06/c06ex01.py
# c06ex01.py

md = {}
soma = 0

for i in range(1, 9):
  print(f"Entre a media do aluno {i}: ", end="")
  md[i] = float(input())
  soma += md[i]

media = soma / 8
print(f"\nMedia geral: {media:.2f}")

input("\nTecle <Enter> para encerrar... ")
